app/templates/macros/toolkit_forms.html

```diff
@@ -126,6 +126,24 @@
   {% endwith %}
 {%- endmacro %}
 
+{% macro checkbox_tree(question_content, service_data, errors, question_number=None, get_question=None) -%}
```

Comment:
Might not hurt to additionally pass through 
`question_number=kwargs.question_number` - won't be needed for now but future-proofing for if we ever use one of these in the declaration or add numbers to service questions.

Although neither of those things feel likely, so could skip it.